api/v2/helpers/bouquets/bouquets_helper.py

```diff
@@ -9,3 +9,11 @@ def query_all_bouquets():
         bouquets = BouquetsSchema(many=True).dump(all_bouquets)
 
     return bouquets
+    
+def query_bouquet(bouquet_id):
+    bouquet = BouquetsModel.query.filter_by(id=bouquet_id).first()
+    if bouquet:
+        return BouquetsSchema(many=False).dump(bouquet)
+
+    return False
```

Comment:
I think it is better to return `None` instead of `False`. It is not good for a function to have more than one return type.